src/services/doppler-sites-client.doubles.ts

```diff
@@ -0,0 +1,34 @@
+import { DopplerSitesClient, PromotionsResult } from './doppler-sites-client';
+
+import { timeout } from '../utils';
+
+export class HardcodedDopplerSitesClient implements DopplerSitesClient {
+  public async getBannerData(lang: string, type: string, page: string): Promise<PromotionsResult> {
+    console.log('getBannerData');
+    try {
+      const response: any = {
+        title: 'mi funcionalidad',
+        functionality: 'mi funcionalidad',
+        description: 'mi descripcion',
+        image_url: 'https://qa.fromdoppler.com/wp-content/uploads/2019/06/login-es.746bf048.png',
+        background_url: 'https://qa.fromdoppler.com/wp-content/uploads/2019/06/violet-yellow.png',
+      };
+      await timeout(1500);
+      return {
+        success: true,
+        value: {
+          title: response.title,
+          functionality: response.functionality,
+          description: response.description,
+          imageUrl: response.image_url,
+          backgroundUrl: response.background_url,
+        },
+      };
+    } catch (error) {
```

Comment:
_Try/catch_ block is not necessary for our dummy double. 

I think that it will be simpler if you remove it and comment the error result to allow test it easily:

```js
public async getBannerData(lang: string, type: string, page: string): Promise<PromotionsResult> {
  console.log('getBannerData');
    await timeout(1500);
    return {
      success: true,
      value: {
        title: 'mi funcionalidad',
        functionality: 'mi funcionalidad',
        description: 'mi descripcion',
        image_url: 'https://qa.fromdoppler.com/wp-content/uploads/2019/06/login-es.746bf048.png',
        background_url: 'https://qa.fromdoppler.com/wp-content/uploads/2019/06/violet-yellow.png',
      }
    };
    // return {
    //   success: false,
    //   error: new Error('Dummy error'),
    // };
}
```